### ISSUE TITLE
Holy Priest Talent (Holy Concentration) Advantageous Bug!!!!

### ISSUE THREAD

**Issue by Zealotry** [Valhalla-Gaming/Tracker#9182]
**Describe the issue you're having**: People were wondering why sometimes holy priests were almost impossible to kill , well guess what: Holy Concentration. It is supposed to activate after a first decast which would ofc lock the caster for 2-4 seconds. The following decasts will lock the target for 70% less time for 20 seconds. Well in our esteemed server, even a melee hit can reset the 20 second timer of this buff. So if hit by multiple targets, free cast with no worries, which is too advantageous in pvp (decasting them would be utterly useless)!!!

**Explain how you expect it work**: Explained and all in the wowhead link

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/spell=221661/holy-concentration

**Comment by forstix** [comment]
yep finally someone notes it  => https://github.com/Valhalla-Gaming/Tracker/issues/8305
